### PR TITLE
FIX: reg summary/dashboard not disabled after submission

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* changes in https://github.com/AngelProtocolFinance/devops/pull/394, not reflected in web-app
* since `EndowmentId` is not returned on `registration-submit` use `Email` instead as indicator that application is approved
https://github.com/AngelProtocolFinance/devops/blob/main/src/AP/lambdas/registration-submit/index.js#L37

* fix 2: check if user signed `FSA` (if applicable) before doc considered complete

TODO:
@jp-mariano, noticed that `EndowmentId` is no longer reflected on  registration record upon approval. Web-app still needs this attribute to redirect users to `profile/{endowId}` on successful registration prompt

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/types/aws/ap/registration.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ export type DoneFSAInquiry = Append<DoneOrgDetails, FSAInquiry, {}>;
 export type DoneDocs = Append<DoneFSAInquiry, TDocumentation, {}>;
 export type DoneBanking = Append<DoneDocs, BankingDetails, {}>;
 
-export type SubmissionDetails = { Email: string; EndowmentId: number };
+export type SubmissionDetails = { Email: string; EndowmentId?: number };
```
Comment (ap-justin) [inline src/types/aws/ap/registration.ts:140]: probably best to add `Approved` state type after `InReview` to make things clearer but this would require more changes - could be in another PR

**Comment by jp-mariano** [comment]
I have added a commit in AngelProtocolFinance/devops#408 that addresses the `EndowmentId` issue and also updated the Lambda `admin-approvals`.

**Review by SovereignAndrey — APPROVED** [review]
